src/test/java/seedu/address/model/internship/StatusTest.java

```diff
@@ -36,12 +36,14 @@ public void isValidStatus() {
         assertFalse(Status.isValidStatus("have not applied"));
 
 
-        // valid status for the default values only
+        // valid status for the default values only. Case-insensitive
         assertTrue(Status.isValidStatus("new"));
         assertTrue(Status.isValidStatus("applied"));
         assertTrue(Status.isValidStatus("assessment"));
         assertTrue(Status.isValidStatus("interview"));
         assertTrue(Status.isValidStatus("offered"));
         assertTrue(Status.isValidStatus("rejected"));
+        assertTrue(Status.isValidStatus("Rejected"));
+        assertTrue(Status.isValidStatus("INTERVIEW"));
```

Comment:
Would it be better if you could also include test cases of mixed capitalisation, such as `APPlied` and `iNTerview`? Such mixed capitalisation is accepted by your code, so it would be nice if you could test them as well.

Reply:
Done!